pkgs/top-level/all-packages.nix

```diff
@@ -5201,12 +5201,20 @@ let
 
   php = php56;
 
-  phpPackages = recurseIntoAttrs (callPackage ./php-packages.nix {});
+  phpPackages = php56Packages;
 
   php55Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
     php = php55;
   });
 
+  php56Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
+    php = php56;
+  });
+
+  php70Packages = recurseIntoAttrs (callPackage ./php-packages.nix {
```

Comment:
@edolstra I can remove `recurseIntoAttrs`, but it was there before so I reused it. Also there have been two packages before already `phpPackages` and `php55Packages`. I have basically adopted this from python packages (IIRC) and they are declared similarly. Also I wonder if we hadn't multiple packages per version, how would I tell nix which version to build a specific package with? Especially if there are multiple PHP versions install on a machine.